src/lib/dateTime.ts

```diff
@@ -0,0 +1,56 @@
+import moment = require('moment')
```

Comment:
Seems like the implementation of the methods in this library can be achieved very quickly with the moment.

Reply:
agree, I've created a task https://cshrdigitalandanalysis.atlassian.net/browse/LPFG-832 to review date/time code

here I just moved functions to a class for better structure